frontend/src/logic/schedule-logic/section-logic.model.ts

```diff
@@ -1,14 +1,8 @@
+import { Sections } from "../providers/schedule-provider.model";
 import { DataRow } from "./data-row";
 
-export interface SectionLogic {
-  sectionKey: string;
-  addDataRow(newRow: DataRow): DataRow[];
-  updateDataRow(rowIndex: number, updateIndexes: number[], newValue: string): DataRow;
-  sectionData: DataRow[];
-}
-
-export abstract class BaseSectionLogic implements SectionLogic {
-  abstract sectionKey: string;
+export abstract class BaseSectionLogic implements BaseSectionLogic {
```

Comment:
O co tu chodzi? Klasa abstrakcyjna implementuje samą siebie

Reply:
Ups. CTRL+SHIFT+F => REPLACE. Usunę to.

Reply:
Usunięte